forge-gui/res/cardsfolder/upcoming/wheel_of_potential.txt

```diff
@@ -0,0 +1,16 @@
+Name:Wheel of Potential
+ManaCost:2 R
+Types:Sorcery
+A:SP$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 3 | SubAbility$ ChooseX | StackDescription$ REP You get_{p:You} gets & you_ | SpellDescription$ You get {E}{E}{E} (three energy counters), then you may pay X {E}.,,,,,,
+SVar:ChooseX:DB$ ChooseNumber | Max$ Count$YourCountersEnergy | ListTitle$ Choose amount for X | SubAbility$ Pay | StackDescription$ None
+SVar:Pay:DB$ Pump | UnlessCost$ Mandatory PayEnergy<X> | UnlessPayer$ You | UnlessSwitched$ True | SubAbility$ Choose | StackDescription$ None
+SVar:Choose:DB$ GenericChoice | Defined$ Player | Choices$ ExileDraw,No | TempRemember$ Chooser | SubAbility$ Effect | StackDescription$ SpellDescription | SpellDescription$ Each player may exile their hand and draw X cards.
```

Comment:
It would be nice if we can do this by the rules instead:
> Players decide and announce in turn order, starting with the active player, whether or not they want to exile their hand. Then, all players who chose to do so exile their hands and draw X cards. Players later in the turn order will know if players before them are exiling their hands, but they don't get to see what those players will exile to help them decide whether to exile their hands themselves.

This issue is also related: #3526